### PR TITLE
Hollow Knight yaml

### PR DESCRIPTION
**Pull request by SnowOfAllTrades**
Hollow Knight async filler yaml

### CODE REVIEW

**Comment by TheT113** [comment]
You'll want to remove the last 3 lines, or else it won't function correctly.

**Comment by SnowOfAllTrades** [comment]
done, thanks

File: games/Hollow Knight.yaml [hunk 1]
```diff
@@ -0,0 +1,103 @@
+Hollow Knight:
+  progression_balancing: 0
```
Comment (t3hf1gm3nt) [inline games/Hollow Knight.yaml:2]: prog balancing is unnecessary since it gets meta'd to off anyway

Comment (SnowOfAllTrades) [reply games/Hollow Knight.yaml:2]: Fair enough, I'll remove the line

**Comment by ScootyPuffJr1** [comment]
You need to add Hollow Knight to the __meta__.yaml file, too.

File: games/Hollow Knight.yaml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  SplitMothwingCloak: 'random'
+  SplitMantisClaw: 'random'
+  CostSanity: off
+  CostSanityHybridChance: 10
+  CostSanityEggWeight: 1
+  CostSanityGrubWeight: 1
+  CostSanityEssenceWeight: 1
+  CostSanityCharmWeight: 1
+  CostSanityGeoWeight: 8
+  local_items:
```
Comment (alwaysintreble) [inline games/Hollow Knight.yaml:96]: uhhh have you test generated with this? the way you have this formatted they're all separate lists which should break. remove the `-` on all but the first line. they still need to be indented though.

**Pull request merged**